### PR TITLE
fix(server): use lower case in error messages

### PR DESCRIPTION
**Pull request by clems4ever**

### CODE REVIEW

File: internal/server/error_handler.go [hunk 1]
```diff
@@ -14,15 +14,15 @@ func autheliaErrorHandler(ctx *fasthttp.RequestCtx, err error) {
 
 	if _, ok := err.(*fasthttp.ErrSmallBuffer); ok {
 		// Note: Getting X-Forwarded-For or Request URI is impossible for ths error.
-		logger.Tracef("Request was too large to handle from client %s. Response Code %d.", ctx.RemoteIP().String(), fasthttp.StatusRequestHeaderFieldsTooLarge)
-		ctx.Error("Request header too large", fasthttp.StatusRequestHeaderFieldsTooLarge)
+		logger.Tracef("request was too large to handle from client %s. Response Code %d.", ctx.RemoteIP().String(), fasthttp.StatusRequestHeaderFieldsTooLarge)
```
Comment (nightah) [inline internal/server/error_handler.go:17]:
```suggestion
		logger.Tracef("request was too large to handle from client %s. response dode %d.", ctx.RemoteIP().String(), fasthttp.StatusRequestHeaderFieldsTooLarge)
```

File: internal/server/error_handler.go [hunk 2]
```diff
@@ -14,15 +14,15 @@ func autheliaErrorHandler(ctx *fasthttp.RequestCtx, err error) {
...
+		ctx.Error("request header too large", fasthttp.StatusRequestHeaderFieldsTooLarge)
 	} else if netErr, ok := err.(*net.OpError); ok && netErr.Timeout() {
 		// TODO: Add X-Forwarded-For Check here.
-		logger.Tracef("Request timeout occurred while handling from client %s: %s. Response Code %d.", ctx.RemoteIP().String(), ctx.RequestURI(), fasthttp.StatusRequestTimeout)
-		ctx.Error("Request timeout", fasthttp.StatusRequestTimeout)
+		logger.Tracef("request timeout occurred while handling from client %s: %s. Response Code %d.", ctx.RemoteIP().String(), ctx.RequestURI(), fasthttp.StatusRequestTimeout)
```
Comment (nightah) [inline internal/server/error_handler.go:21]:
```suggestion
		logger.Tracef("request timeout occurred while handling from client %s: %s. response code %d.", ctx.RemoteIP().String(), ctx.RequestURI(), fasthttp.StatusRequestTimeout)
```

File: internal/server/error_handler.go [hunk 3]
```diff
@@ -14,15 +14,15 @@ func autheliaErrorHandler(ctx *fasthttp.RequestCtx, err error) {
...
+		ctx.Error("request timeout", fasthttp.StatusRequestTimeout)
 	} else {
 		// TODO: Add X-Forwarded-For Check here.
-		logger.Tracef("An unknown error occurred while handling a request from client %s: %s. Response Code %d.", ctx.RemoteIP().String(), ctx.RequestURI(), fasthttp.StatusBadRequest)
-		ctx.Error("Error when parsing request", fasthttp.StatusBadRequest)
+		logger.Tracef("an unknown error occurred while handling a request from client %s: %s. Response Code %d.", ctx.RemoteIP().String(), ctx.RequestURI(), fasthttp.StatusBadRequest)
```
Comment (nightah) [inline internal/server/error_handler.go:25]:
```suggestion
		logger.Tracef("an unknown error occurred while handling a request from client %s: %s. response code %d.", ctx.RemoteIP().String(), ctx.RequestURI(), fasthttp.StatusBadRequest)
```

**Review by nightah — CHANGES_REQUESTED** [review]
Capitalizing just this element looks a bit odd.
WDYT?

**Comment by james-d-elliott** [comment]
> 
> 
> Capitalizing just this element looks a bit odd.
> WDYT?

Probably a good idea if we make everything lowercase except maybe acronyms or other reasonable exceptions?

**Comment by clems4ever** [comment]
I'm fine with both propositions. I've committed the suggestions.

**Review by nightah — CHANGES_REQUESTED** [review]
For consistency lets not lower case log messages.

**Review by james-d-elliott — APPROVED** [review]
LGTM

**Pull request merged**